arangosh/Shell/ClientFeature.cpp

```diff
@@ -86,23 +90,38 @@ void ClientFeature::collectOptions(std::shared_ptr<ProgramOptions> options) {
                      "for a password",
                      new StringParameter(&_password));
 
+  if (_allowJwtSecret) {
+    options->addHiddenOption(
+        "--server.ask-jwt-secret",
+        "if this option is specified, the user will be prompted "
+        "for an jwt secret. This option is not compatible with "
+        "--server.password. If specified it will be used for all "
+        "connection - even when a new connection  to another server is "
+        "created",
+        new BooleanParameter(&_askJwtSecret));
+  } else {
+    options->addObsoleteOption("--server.ask-jwt-secret",
```

Comment:
why add it as an obsolete option and not omit it entirely for all client tools that do not support it?